accounts-fiware_login_button.css

```diff
@@ -1,3 +1,3 @@
-#login-buttons-image-imgur {
+#login-buttons-image-fiware {
     background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAABAklEQVQ4jZ2RMU7DMBSGPwcnjauEuEYgKKgjI+IETJyhC+foSE/SBU7BJRALE2qZw4hgSEliMxVV1Ena/pOH933P7z0xebp9sM5eskdUGr5K6+yVw17vCseJpHcYEOzTOU4kKgsRQrQLBtGQQTRshIFmQRQoxqMp49E9UaC8MIBsEpR2SV7M/94+uFXgsCy+ngHoJYEXbhUciJCbkzu0zpjlL1Tux1vXuAOBQGuNMUcINjt3ClQaYYzxfrtTECeSfsPM/7Oxg9W2a2pKuwSgpt5OsH4q6yoe3ycAWFd1C3x3zovFdiOcmnNS3e8sXk/lKj7LD6TW2dvZ8cVO8CpF/T3/BaxpQTa7TfPPAAAAAElFTkSuQmCC);
```

Comment:
@mauriciovieira i need the fiware logo to put in package, where i can find a nice image of fiware?